lib/takeaway.rb

```diff
@@ -0,0 +1,70 @@
+require 'twilio-ruby'
+
+# put your own credentials here
+account_sid = 'AC71ede767f3f69bf43718d8094b244800'
+auth_token = '88ac7f43909c00b5b8e9a32d5b554c35'
+
+# set up a client to talk to the Twilio REST API
+@client = Twilio::REST::Client.new account_sid, auth_token
+
+class Menu
+  def initialize
+    @list_of_dishes = []
+  end
+  def list_of_dishes
+    @list_of_dishes
+  end
+end
+
+class Meal
+  attr_accessor :name, :price
+  def initialize(name, price)
+    @name = name
+    @price = price
+  end
+
+  def list_price
+    @price
+  end
+
+  def list_name
+    @name
+  end
+end
+
+def send_a_text
+  @client.messages.create(
+    from: '+13605295607',
+    to: '+447808274842',
+    body: 'Test'
+  )
+  puts ("tried to send a text")
+end
+
+def show_menu
+  puts("Available dishes: ")
+  puts(m.list_of_dishes)
+  puts("To make an order, specify the meal, followed by the quantity")
+  make_order
+end
+
+def make_order
+  order = []
+  p("name of meal:")
+  ordername = gets.chomp
+  p("quantity:")
+  orderquan = gets.chomp.to_i
+  order << ordername
+  order << orderquan
+  process_order(order)
+end
+
+def process_order(order)
+end
```

Comment:
All these methods - `send_a_text`, `show_menu`, `make_order` - should live on a class. Think about what classes best suit the responsibilities of these methods.